patchwork-model-loader/src/main/java/net/patchworkmc/impl/modelloader/AbstractModelLoader.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Minecraft Forge, Patchwork Project
+ * Copyright (c) 2016-2020, 2019-2020
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.patchworkmc.impl.modelloader;
+
+import java.util.Set;
+
+import net.minecraft.util.Identifier;
+
+public interface AbstractModelLoader {
```

Comment:
Abstract implies a class.
Maybe something like `SpecialModelProvider`?

Reply:
An `I` prefix would be okay if my name doesn't make sense because more code would needed to be added here later.

Reply:
fixed in the latest commit